blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -1,9 +1,9 @@
 import keymirror from 'keymirror';
-import { fetch as smartFetch, paginate, applyFetchMarkers } from '../util/smart-fetch';
+import { applyFetchMarkers, fetch as smartFetch, paginate } from '../util/smart-fetch';
```

Comment:
The `smart-fetch` utils aren't really being used in elsewhere in the app, in favor of the mobx-based `Pager` implementation that @imeredith worked on. I feel like the scope of this PR should be expanded to include porting Test Results to mobx so we can avoid adding more code that needs to be ported later :/